### ISSUE TITLE
DataException error on refresh

### ISSUE THREAD

**Issue by DunnesH** [digitalec/deemon#22]
Running version 1.2, Windows 10, getting this error every time I run deemon refresh
```
Refreshing artists...  12/1462 [                                        ]   1%
Traceback (most recent call last):
  File "c:\users\main\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\main\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\main\AppData\Local\Programs\Python\Python39\Scripts\deemon.exe\__main__.py", line 7, in <module>
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\deemon\__main__.py", line 4, in main
    cli.run()
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\deemon\cli.py", line 169, in refresh_command
    refresh.refresh()
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\deemon\app\refresh.py", line 118, in refresh
    albums = self.dz.api.get_artist_albums(artist["id"])
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\deezer\api.py", line 78, in get_artist_albums
    return self.api_call(f'artist/{str(artist_id)}/albums', {'index': index, 'limit': limit})
  File "c:\users\main\appdata\local\programs\python\python39\lib\site-packages\deezer\api.py", line 51, in api_call
    if result_json['error']['code'] == 800: raise DataException(f"DataException: {method} {result_json['error']['message'] if 'message' in result_json['error'] else ''}")
deezer.api.DataException: DataException: artist/A-GON/albums no data
```
I've tried removing A-GON from the monitoring list but I still get the same error

**Comment by digitalec** [comment]
Ok I will look into it this afternoon.

You ran this?
`$ deemon monitor --remove "A-GON"`

Check and see if the artist is still in your monitoring list via `deemon show -a`.

**Comment by digitalec** [comment]
I just tried monitoring and refreshing with this artist and had no issue. Can you try again?

**Comment by DunnesH** [comment]
Yes, ```deemon monitor --remove "A-GON"``` exactly as you typed and it returns ```No longer monitoring A-GON```, but the artist does still show up in ```deemon show -a``` and running the remove command again returns```Artist 'A-GON' not found```. 

I believe the problem is that deemon didn't like how I imported the artists. I used a single txt with a mix of IDs and artist names, and ran ```deemon import -i "artists.txt"``` and then just ```deemon import "artists.txt"```. The refresh says "12/1462" but there are only 996 artists in the text file. I guess this added some of the artists twice.

I had to uninstall+reinstall, but after splitting the names & IDs into two documents and then importing again separately, it works fine now. Thanks for the help!
